pkg/healthcheck/healthcheck.go

```diff
@@ -1776,6 +1776,13 @@ func (hc *HealthChecker) fetchWebhookCaBundle(ctx context.Context, webhook strin
 	return caBundle, nil
 }
 
+// FetchTrustBundle retrieves the ca-bundle from the config-map linkerd-identity-trust-roots
+func FetchTrustBundle(ctx context.Context, kubeAPI k8s.KubernetesAPI, controlPlaneNamespace string) (string, error) {
```

Comment:
Nice! This is exactly what I had in mind :)

Reply:
🙃